app/konnect/dev-portal/access-and-approval/add-teams.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Add Developer Teams from Identity Providers
+content_type: how-to
+---
+
+In the {{site.konnect_short_name}} Dev Portal, you can import existing developer teams from a third-party identity provider (IdP) and map their permissions to elements in {{site.konnect_short_name}}. This allows you to import developers automatically instead of manually copying over every single team of developers from your IdP to {{site.konnect_short_name}}.
+
+This guide explains how to map the permissions, including scopes and claims, from your group of developers in your IdP to your organization's team in {{site.konnect_short_name}}. Although this guide uses Okta, Azure, and Auth0 as examples, you can use any IdP that conforms to OIDC standards. 
+
+## Prerequisites
+An IdP configured in Konnect (OIDC or DCR) for [application registration](konnect/dev-portal/applications/enable-app-reg/)
+
+## Create developer teams in your IdP
+
+<!-- do these IdPs need the Kong callback URl added to them to be properly configured? Or is this just a one-way pull from the IdP to Konnect, so it doesn't need any information from Konnect?-->
+
+{% navtabs %}
+{% navtab Okta %}
+In Okta, create a [new group of developers](https://help.okta.com/asa/en-us/content/topics/adv_server_access/docs/setup/create-a-group.htm) that you want to map to {{site.konnect_short_name}}. Be sure to configure the team settings and attributes as needed.
+{% endnavtab %}
+{% navtab Azure %}
+In Azure, create a [new team of developers](https://learn.microsoft.com/azure/devops/organizations/settings/add-teams?view=azure-devops&tabs=preview-page) that you want to map to {{site.konnect_short_name}}.
+{% endnavtab %}
+{% navtab Auth0 %}
+In Auth0, create a [new team of developers](https://auth0.com/docs/get-started/tenant-settings/auth0-teams) that you want to map to {{site.konnect_short_name}}.
+{% endnavtab %}
+{% endnavtabs %}
```

Comment:
@c3pko For the docs that I linked, could you check and make sure that these are the right instructions for creating teams in the respective IdPs? I haven't worked with any of these before, so I wasn't sure if these were the right docs to link or if there's any other instructions/advice we need to provide.

Reply:
The links look good. Are we sure the mapping of teams in the different IdPs requires the same steps/there aren't any IdP specific nuances?